en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/configure-logs-overview.md

```diff
@@ -0,0 +1,16 @@
+# Overview
+
+Logging in choreo connect is really important when debugging issues in a short period of time and looking into the underlying details of how each component works.  Choreo connect provides facilities for getting the logs in various formats (JSON, Plain text), various output methods and various logging levels.
+
+In addition to these, the adapter and enforcer components have been defined with a set of error codes for each of the components’ error logs, in order to narrow down the details of errors. Following are the error codes ranges for each of the components.
+
```

Comment:
add more details (i.e. pointers to the other sections like troubleshooting, enforcer, adapter & router sections..)

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/5357/commits/4681da56484b9296768a350aab44df5936c04cb7